### ISSUE TITLE
Keep consistency with --dev option

### ISSUE THREAD

**Issue by Seb33300** [symfony/symfony-docs#10933]
Hello,

I am not very comfortable with command line and I was wondering why in some cases you write:
```
composer require --dev bundle-name
```
And on some other cases:
```
composer require bundle-name --dev
```

It also concern yarn command lines: 
https://github.com/symfony/symfony-docs/search?p=1&q=--dev&unscoped_q=--dev

Since it could be confusing for beginners or non-friendly command-line people like me, I suggest to keep consistency and always use the same order.

I will suggest to use `composer require --dev bundle-name` syntax since it sounds closer to the related composer `require-dev` option.

I can provide a pull request if 👍

**Comment by Seb33300** [comment]
@javiereguiluz was faster

**Comment by javiereguiluz** [comment]
@Seb33300 I'm sorry for not having let you prepare this PR. The reason is that this kind of PRs usually generate conflicts so I wanted to give it a try to estimate the conflicts. It was much better than expected, so I created the PR. Thanks.

**Comment by javiereguiluz** [comment]
Fixed by #10938.
